mantidproject/mantid
# Update WAND² IDF

Update WAND² IDF to round s2 and detz; also invert pixel indexing along wire so hopefully it's now correct.

The pixel mapping will probably change again before we start up again but hopefully it is correct for now.

**To test:**
Look at the diff of the IDF.

Try loading some data *e.g.* `ws = LoadEventNexus('/HFIR/HB2C/IPTS-7776/nexus/HB2C_6500.nxs.h5')` and see that it looks correct.

*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
